log-insights/setup/opentelemetry.mdx

```diff
@@ -31,18 +78,122 @@ In values.yaml file, update the section of `service`:
 </CodeBlock>
 
 This will create a service with the name `pganalyze-collector-otel-service`.
-Assuming you use [Fluent Bit](https://fluentbit.io/) as a telemetry agent,
-you can specify the created service as an OpenTelemetry HTTP endpoint, using the
-[OpenTelemetry output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/opentelemetry):
+
+To create multiple OpenTelemetry endpoints, you can add multiple port entries under the `ports` section:
+
+<CodeBlock language="yaml">
+{`service:
+  create: true
+  name: pganalyze-collector-otel-service
+  type: ClusterIP
+  port: 4318
+  targetPort: 4318
```

Comment:
This example might be a bit confusing, as this contains the `port` way of setting and `ports` way of setting. I'd simply remove above two lines for the multiple port entries example here.